server/server.js

```diff
@@ -1490,6 +1562,19 @@ async function restartMonitor(userID, monitorID) {
     return await startMonitor(userID, monitorID);
 }
 
+async function restartMonitors(userID) {
```

Comment:
This function creates another set of monitors, because it does not stop current monitors.

But as the proxy config of monitors should be updated without restarting, this function is not needed anyway in my opinion.